sdk/nodejs/tests/provider/experimental/analyzer.spec.ts

```diff
@@ -209,6 +211,7 @@ describe("Analyzer", function () {
                     outMapOfStrings: { type: "object", additionalProperties: { type: "string" } },
                     outMapOfNumbers: { type: "object", additionalProperties: { type: "number" } },
                     outMapOfBooleans: { type: "object", additionalProperties: { type: "boolean" } },
+                    outMapOptionalStrings: { type: "object", additionalProperties: { type: "string", optional: true } },
```

Comment:
Isn't this wrong? The entire dictionary is optional, not its string values?

Reply:
Yeah, it's wrong (and there were some more things wrong with outputs and optional types), and the test was broken, I just had too much confidence and forgot to run it locally before submitting the PR :sweat_smile: